pkg/check/gc/chunk_notfound.go

```diff
@@ -0,0 +1,67 @@
+package gc
```

Comment:
Add copyright header.

Reply:
there are no copyright headers across the entire repo (at all). should i really do this as part of this pr?

Reply:
Ah, sorry, I was reviewing with bee in mind.